### PR TITLE
Fix flashlights not working in space

### LINKED ISSUE #2090: flashlight does not work in space
#### Description of issue
flashlight does not work in space
its turned on:
изображение

#### Difference between expected and actual behavior
flashlight should make light

#### Steps to reproduce
1) go to the space tiles or lattice
2) turn it on

#### Issue bingo

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/NebulaSS13/Nebula/issues)

### PR DESCRIPTION
**Pull request by out-of-phaze**
## Description of changes
Fixes #2090. Currently targeting dev because I'm unsure if this change is valid or not, will probably retarget stable if Lohikar says it's okay.

dreamseeker_Vm1T7dr6jR

## Why and what will this PR improve
Fixes #2090.

### CODE REVIEW

File: code/modules/lighting/lighting_source.dm [hunk 1]
```diff
@@ -358,7 +358,7 @@
 			if ((DETERMINANT(limit_a_x, limit_a_y, test_x, test_y) > 0) || DETERMINANT(test_x, test_y, limit_b_x, limit_b_y) > 0)
 				continue
 
-		if (TURF_IS_DYNAMICALLY_LIT_UNSAFE(T) || T.light_source_solo || T.light_source_multi)
+		if (TURF_IS_DYNAMICALLY_LIT_UNSAFE(T) || T.corners || T.light_source_solo || T.light_source_multi)
```
Comment (Lohikar) [inline code/modules/lighting/lighting_source.dm:361]: I'm pretty sure the correct fix is:
```suggestion
		if (TURF_IS_DYNAMICALLY_LIT_UNSAFE(T) || T.ambient_light || T.light_source_solo || T.light_source_multi)
```

Comment (out-of-phaze) [reply code/modules/lighting/lighting_source.dm:361]: corners is wrong, but ambient_light doesn't cover the case where space ambience/starlight is disabled.

**Comment by MistakeNot4892** [comment]
Per Discord, this issue will need to be fixed on stable by Lohikar, so closing this one.

File: code/modules/lighting/lighting_source.dm (hunk 1 above)
Comment (out-of-phaze) [reply code/modules/lighting/lighting_source.dm:361]: wait. no, in that case wouldn't corners be right?

**Pull request closed without merging**